src/Features/CSharp/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.cs

```diff
@@ -104,7 +104,13 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             if (local != null)
             {
                 var findReferencesResult = await SymbolFinder.FindReferencesAsync(local, document.Project.Solution, cancellationToken).ConfigureAwait(false);
-                var locations = findReferencesResult.Single(r => Equals(r.Definition, local)).Locations;
+                var referencedSymbol = findReferencesResult.SingleOrDefault(r => Equals(r.Definition, local));
```

Comment:
why did this change to single-or-default?

Reply:
That line threw an exception in #43666 and crashed the refactoring provider. I opened the original issue but haven't been able to repro it since then, so I'm unable to figure out the root cause, although I thought this might be a good way to at least prevent the crash from occurring.

Reply:
let's undo this.  i don't like this offhand change without a clear thing we know it's fixing.

Reply:
I just hit this same issue in VB code. I don't know exactly what it's fixing, but we certainly need to keep it.